ext/crypto/decrypt.rs

```diff
@@ -73,12 +70,36 @@ pub enum DecryptAlgorithm {
   },
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum DecryptError {
+  #[error(transparent)]
+  General(#[from] SharedError),
+  #[error(transparent)]
+  Pkcs1(#[from] rsa::pkcs1::Error),
+  #[error("Decryption failed")]
```

Comment:
All other error strings start as a lowercase. Is that supposed to be lowercase too?

Reply:
I am planning to clean up the messages a bit on a follow-up; for now i just moved the strings, so this is like it was before. I agree this should be more consistent, but wont do that in this PR